lib/core/mqtt.cpp

```diff
@@ -0,0 +1,174 @@
+#include "mqtt.hpp"
+
+#include <rapidjson/writer.h>
+
+#include "rapidjson/stringbuffer.h"
+
+namespace hyped::core {
+
+std::optional<std::shared_ptr<Mqtt>> Mqtt::create(ILogger &logger,
+                                                  const std::string &id,
+                                                  const std::string &host,
+                                                  const std::uint16_t port)
+{
+  std::string address = "tcp://" + host + ":" + std::to_string(port);
+  mqtt::connect_options connection_options;
+  connection_options.set_keep_alive_interval(1);
```

Comment:
What unit is this, and might we want to make it part of the constructor and include in a config file?

Reply:
Appears to be longest time no messages pass in seconds.
```
/**
	 * Sets the "keep alive" interval.
	 * This is the maximum time that should pass without communications
	 * between client and server. If no messages pass in this time, the
	 * client will ping the broker.
	 * @param keepAliveInterval The keep alive interval in seconds.
	 */
```
Does 1 second seem like a good amount of time?

Reply:
constant to be added for the keep alive interval